locales/base/translation.json

```diff
@@ -2073,6 +2073,10 @@
     "notEnoughBalanceForGasWarning": {
       "title": "You need more {{feeTokenSymbol}} for gas",
       "description": "Add {{feeTokenSymbol}} to cover your gas fees or adjust the amount you are sending"
+    },
+    "feeEstimationError": {
+      "title": "Error estimating network fee",
+      "description": "There was an error estimating the network fee. Please try again later."
```

Comment:
This error message is a bit generic, but this could appear in case of any error while calculating the fees.
In the MNEC case, probably we want to show a more precise message indicating the problem won't be fixed?
If we want that, I'll address it in another PR.